package.json

```diff
@@ -55,7 +56,6 @@
     "mocha": "3.4.2",
     "ngrok": "2.2.10",
     "node-watch": "0.5.4",
-    "semver": "5.3.0",
```

Comment:
@BrunoBernardino Please, check @xavdid's first comment on `package.json` in [this PR](https://github.com/zapier/zapier-platform-cli/pull/106) as regards making `semver` a dev dependency.